kernel/mlx_ethernet/src/completion_queue.rs

```diff
@@ -13,8 +16,11 @@ use alloc::boxed::Box;
 use memory::{PhysicalAddress, MappedPages};
 use owning_ref::BoxRefMut;
 use num_enum::TryFromPrimitive;
+
+#[allow(unused_imports)]
 use crate::{
     *,
+    command_queue::CommandOpcode,
     work_queue::WQEOpcode
 };
```

Comment:
this import statement is suspect. Either use `crate::*` or specify the exact items. The glob is only preferred if there are dozens of things you need to import